src/page.ts

```diff
@@ -114,6 +124,39 @@ export class Page extends EventTarget {
       );
     });
 
+    if (options?.sandbox) {
+      const { sandbox, sandboxPrompt } = options;
+      this.#celestial.Fetch.enable({});
```

Comment:
This creates a race condition (which is unlikely to matter in IRL code, but I've encountered before unfortunately). Could you move the fetch enabling to the browser's `newPage` method? I believe there should be some other enablings there.

Reply:
Ok I'll move it this to `browser.ts` instead 👍